src/main/java/core/basesyntax/JavaStreamApi.java

```diff
@@ -38,7 +47,10 @@ public Long elementCount(List<String> elements, String element) {
      * Найти элемент в коллекции равный `element` или кинуть ошибку NoSuchElementException</p>
      **/
     public String findElement(List<String> elements, String element) {
-        return null;
+        return elements.stream()
+                .filter(s -> s == element)
+                .findFirst()
+                .orElseThrow(NoSuchElementException::new);
```

Comment:
orElseThrow() throws NoSuchElementException by itself, so we don't need to create new exception